student.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
```

Comment:
- [ ] OPTIONAL: Nice work. Kindly use `require_relative` here. It is recommended to import modules in the project local directory.

Reply:
Thank you for letting me know. I made the changes.